common/src/main/java/org/apache/celeborn/common/meta/FileInfo.java

```diff
@@ -106,8 +110,17 @@ public synchronized long getLastChunkOffset() {
     return chunkOffsets.get(chunkOffsets.size() - 1);
   }
 
-  public synchronized long getFileLength() {
-    return chunkOffsets.get(chunkOffsets.size() - 1);
+  public long getFileLength() {
+    return getBytesFlushed();
+  }
+
+  public long getBytesFlushed() {
```

Comment:
I think we can delete this method and just use getFileLength

Reply:
getBytesFlushed has been removed